internal/langserver/handlers/command/module_calls_test.go

```diff
@@ -0,0 +1,95 @@
+package command
+
+import (
+	"encoding/json"
+	"reflect"
+	"testing"
+
+	"github.com/hashicorp/terraform-ls/internal/terraform/datadir"
+)
+
+func Test_parseModuleRecords(t *testing.T) {
+	type args struct {
+		records []datadir.ModuleRecord
+	}
+	tests := []struct {
+		name string
+		args args
+		want []moduleCall
```

Comment:
```suggestion
	tests := []struct {
		name string
		records []datadir.ModuleRecord
		want []moduleCall
```

Nitpick, but I think there is no need for that extra type definition?

Reply:
This is from the neat Go extension generated test cases, and works well with multiple parameter functions but not so well with single parameters.